source/dem/dem.cc

```diff
@@ -671,6 +671,44 @@ DEMSolver<dim>::write_output_results()
     }
 }
 
+template <int dim>
+void
+DEMSolver<dim>::write_forces_torques_output_results()
+{
+  std::map<unsigned int, Tensor<1, dim>> force_on_walls;
+  std::map<unsigned int, Tensor<1, dim>> torque_on_walls;
+  force_on_walls=pw_contact_force_object->get_force();
+  torque_on_walls=pw_contact_force_object->get_torque();
+  if ( (parameters.forces_torques.force_torque_display_method ==
+      Parameters::Lagrangian::ForceTorqueOnWall<
+        dim>::ForcesAndTorquesDisplay::terminal) ||
+      (parameters.forces_torques.force_torque_display_method ==
+      Parameters::Lagrangian::ForceTorqueOnWall<
+        dim>::ForcesAndTorquesDisplay::both) )
```

Comment:
This would be changed by the proposed modification above. Always assume you are writing to file.